ctsit/imagemap
# Add noneoftheabove compatibility

Closes #72 

`@NONEOFTHEABOVE` tags were added to `mbody` and `66swollen_68tender_joint_count` for testing. No saving of data is needed.

To test:
1. select a few regions in one of the imagemaps
2. select the region at the top of the imagemap that suggests it'll delete the other options
3. click confirm in the dialog box (note that cancelling will disable the feature of this PR entirely)
4. observe that previously selected regions are deselected

I am appending logic to a core function rather than redefining it entirely to avoid compatibility issues. This has caused a couple weaknesses in this update, though they only apply to the imagemap UI, not the actual stored data:
- Cancelling the dialog box disables this adaptation entirely
- The `@NONEOFTHEABOVE` tagged option will not be deselected when selecting new entries
Attempts to rectify these edge cases are taking significantly more time than solving the stated problem so I'm leaving them in for now
